components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
     {
       "effect": "ALLOW",
       "actions": [
-        "iam:users:list"
-      ]
+        "iam:users:list",
+        "iam:users:get"
+      ],
+      "projects": [
+        "*"
+      ],
     },
     {
       "effect": "ALLOW",
       "role": "editor"
     }
+  ],
+  "projects": [
+    "admin-group-1",
+    "it-support",
   ]
 }
 ```
```

Comment:
This API reference, of course, needs to list all the properties for a given resource (like a policy in this instance). As a first pass, I chose to do this with a sample like the one just above. I am not sure that is adequate though--should we also have a table that lists each property and what it means?